hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/SupportedDatabasesTest.java

```diff
@@ -57,8 +59,19 @@ public void testUpsertDialectSupported() {
 
         when(jdbcTable.sqlDialect()).thenReturn(mysqlSqlDialect);
 
-        boolean result = UpsertBuilder.isUpsertDialectSupported(jdbcTable);
+        boolean result = SupportedDatabases.isDialectSupported(jdbcTable);
         assertTrue(result);
     }
 
+    @Test
+    public void logByProductName_supportedDB() {
+        boolean newDB = SupportedDatabases.logOnceByProductName("mysql");
```

Comment:
You should invoke it twice at least

Reply:
Ok. I will invoke logOnceByProductName() method twice